backend/tests/db/test_tests.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from ipaddress import IPv4Address
+
+import pytest
+from faker import Faker
+from sqlalchemy.orm import Session as OrmSession
+
+from mirrors_qa_backend import db
+from mirrors_qa_backend.db import models
+from mirrors_qa_backend.enums import StatusEnum
+
+
+@pytest.mark.num_tests(1)
+def test_get_test(dbsession: OrmSession, tests: list[models.Test]):
+    test = tests[0]
+    result = db.tests.get_test(dbsession, test.id)
+    assert result is not None
+    assert result.id == test.id
+
+
+@pytest.mark.parametrize(
+    ["worker_id", "country", "status", "expect"],
+    [
+        (None, None, None, True),
+        ("worker_id", None, None, False),
+        (None, "country", [StatusEnum.PENDING], False),
+        (None, None, list(StatusEnum), True),
+        ("worker_id", "country", [StatusEnum.PENDING], False),
+    ],
+)
+def test_basic_filter(
+    dbsession: OrmSession,
+    worker_id: str | None,
+    country: str | None,
+    status: list[StatusEnum] | None,
+    expect: bool,  # noqa
```

Comment:
```suggestion
    expected: bool,  # noqa
```

Reply:
What's this `# noqa` for? 
I'd prefer explicit exclusion or event specific exclusions in per-rile excludes in pyproject